src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,44 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(health = 100, name) {
+    this.health = health;
+    this.name = name;
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
-}
+  // write your code here0
+  constructor(name, health = 100, hidden = false) {
+    super(name, health, hidden);
+    this.hidden = hidden;
+    this.health = health;
+    Animal.alive.push(this);
+  }
```

Comment:
[CODE LOGIC] You are passing `hidden` to the `super()` method, but the `Animal` constructor does not accept a `hidden` parameter. You should remove `hidden` from the `super()` method.